tests/templates/test_tensornetworks/test_MERA.py

```diff
@@ -0,0 +1,447 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
The tests are great! My only worry is they may be a bit too heavy. How long do they take to run?

Reply:
0.17 seconds on my machine.